packages/realm/src/app-services/ApiKeyAuthClient.ts

```diff
@@ -16,7 +16,8 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
-import { User, binding } from "../internal";
+import { SyncUser } from "../binding";
+import { User, binding, Realm } from "../internal";
```

Comment:
I'd rather that the internal code doesn't rely on the namespaces, as they might very well go away at some point.

```suggestion
import { BSON, User, binding } from "../internal";
```